parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -107,6 +107,14 @@ write_files:
     RemainAfterExit=yes
     ExecStart=/usr/local/bin/health-monitor.sh container-runtime
 
+- path: /etc/apt/preferences
+  permissions: "0644"
+  owner: root
+  content: |
+    Package: walinuxagent
+    Pin: version 2.2.32.2
```

Comment:
is that the version that is currently in VHD? or the version we are currently being upgraded to ?

Reply:
The version in the VHD